src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -48,10 +46,23 @@ public class Arm extends SubsystemBase {
      */
     private final TalonFX[] motors = new TalonFX[ArmConstants.MOTOR_IDS.length];
     // DCMotor model is 4 Kraken X60
-    private static final DCMotor motorModel = DCMotor.getKrakenX60(ArmConstants.MOTOR_IDS.length);
+    protected static final DCMotor motorModel = DCMotor.getKrakenX60(ArmConstants.MOTOR_IDS.length);
```

Comment:
Why did you make this protected? Nothing extends Arm.

Reply:
Code will not compile when private. Package private should also work.